lib/streamwrapper.php

```diff
@@ -431,16 +432,17 @@ public function dir_closedir() {
 	public function dir_rewinddir() {
 		$this->boolCall(function () {
 			$this->objectIterator = null;
-			$this->dir_opendir($this->openedPath, null);
+			$this->dir_opendir($this->openedPath, '');
```

Comment:
`dir_opendir` expects a string in the second parameter, so keep it happy.
(even though in practice in the implementation, the second parameter is not even used)